spec/mongoid/geospatial/fields/point_spec.rb

```diff
@@ -83,7 +83,7 @@
     end
 
     it 'should have an ActiveModel symbol accessor' do
-      expect(bar[:location]).to eq([3, 2])
+      expect(bar[:location].to_a).to eq([3, 2])
```

Comment:
This seems like a regression, I'm trying to figure out how to make this work with newer dependencies.